polynote-kernel/src/main/scala/polynote/kernel/lang/scal/ScalaSource.scala

```diff
@@ -135,15 +135,15 @@ class ScalaSource[G <: Global](
           case tree: global.ValDef =>
             if (!tree.mods.hasFlag(global.Flag.PRIVATE) && !tree.mods.hasFlag(global.Flag.PROTECTED)) {
               // make a lazy val accessor in the companion
-              (moduleTrees :+ q"lazy val ${tree.name} = INSTANCE.${tree.name}", classTrees :+ tree)
+              (moduleTrees :+ atPos(tree.pos)(q"lazy val ${tree.name} = INSTANCE.${tree.name}"), classTrees :+ tree)
             } else {
               (moduleTrees, classTrees :+ tree)
             }
           case tree: global.DefDef =>
             // make a proxy method in the companion
             val typeArgs = tree.tparams.map(tp => global.Ident(tp.name))
             val valueArgs = tree.vparamss.map(_.map(param => global.Ident(param.name)))
-            (moduleTrees :+ tree.copy(rhs = q"INSTANCE.${tree.name}[..$typeArgs](...$valueArgs)"), classTrees :+ tree)
+            (moduleTrees :+ atPos(tree.pos)(tree.copy(rhs = q"INSTANCE.${tree.name}[..$typeArgs](...$valueArgs)")), classTrees :+ tree)
```

Comment:
do these `atPos` calls fix the line number issue we discussed this morning?

Reply:
Yep! Was worried that Scala would flip out when the positions didn't make sense, but in this case it didn't seem to mind 🤷‍♂️